.changeset/tsconfig-dangerously.md

```diff
@@ -0,0 +1,12 @@
+---
+'sku': minor
+---
+
+`srcPaths` no longer affects `tsconfig.json#include`. Instead, you can use the [`dangerouslySetTSConfig`][dangerous] option to have more control over which files are included in the TypeScript compilation process.
```

Comment:
Just worth clarifying this, since we don't actually use `tsc` for compilation.
```suggestion
`srcPaths` no longer affects `tsconfig.json#include`. Instead, you can use the [`dangerouslySetTSConfig`][dangerous] option to have more control over which files are included in the typechecking process.
```